crates/ra_cargo_watch/src/conv/snapshots/ra_cargo_watch__conv__test__snap_clippy_pass_by_ref.snap

```diff
@@ -58,44 +58,26 @@ expression: diag
                         },
                         message: "lint level defined here",
                     },
+                    DiagnosticRelatedInformation {
```

Comment:
Not sure what this is.

Reply:
It's an additional line of feedback in the issue panel. Because it's not added as a fix, we show it as a hint in there instead